### PR TITLE
Add a command to list the locales players are using

### PR DESCRIPTION
**Pull request by cswhite2000**

### CODE REVIEW

**Comment by Matic0B** [comment]
idk if we are using "/pl" for anything, might be nice to have as an alias

**Comment by RainoBoy97** [comment]
Bukkit uses `/pl` as an alias for `/plugins` already

File: Commons/bukkit/src/main/java/tc/oc/commons/bukkit/commands/MiscCommands.java [hunk 1]
```diff
@@ -91,6 +91,36 @@ public void listPlayerVersions(final CommandContext args, final CommandSender se
...
+            max = 1
+    )
+    @CommandPermissions("ocn.locale")
+    public void listPlayerLocales(final CommandContext args, final CommandSender sender) throws CommandException {
+        Audience audience = audiences.get(sender);
+        if (args.hasFlag('a')) {
+            Map<String, Integer> playerLocaleMap = new HashMap<>();
+            userStore.stream().forEach(player -> {
+                String locale = player.getLocale();
+                playerLocaleMap.put(locale, playerLocaleMap.getOrDefault(locale, 0) + 1);
```
Comment (AustinLMayes) [inline Commons/bukkit/src/main/java/tc/oc/commons/bukkit/commands/MiscCommands.java]: Atomic integers would be more efficient

Comment (Pablete1234) [reply Commons/bukkit/src/main/java/tc/oc/commons/bukkit/commands/MiscCommands.java]:
```
Map<String, Integer> playerLocaleMap = userStore.stream().collect(Collectors.groupingBy(Player::getLocale, Collectors.counting()));```

**Comment by PotatoStealer** [comment]
make this into an analytic component instead. should reflect the most used locale.

**Pull request merged**